cmd/loop/liquidity.go

```diff
@@ -173,6 +174,131 @@ func setRule(ctx *cli.Context) error {
 	return err
 }
 
+var setParamsCommand = cli.Command{
+	Name:        "setparams",
+	Usage:       "update the parameters set for the liquidity manager",
+	Description: "Updates the parameters set for the liquidity manager.",
+	Flags: []cli.Flag{
+		cli.Float64Flag{
+			Name: "maxswapfee",
+			Usage: "the maximum ratio of swap volume we are " +
+				"willing to pay in server fees (eg, 0.01 = " +
+				"1%).",
+		},
+		cli.Float64Flag{
+			Name: "maxroutingfee",
+			Usage: "the maximum ratio of off-chain payment " +
+				"volume that are are willing to pay in " +
+				"routing fees (eg, 0.01 = 1%).",
```

Comment:
Does this also apply to the prepay? With small fractions, it may get problematic?

Reply:
Separated these out, but I think we should be ok now that prepay on mainnet is 20k? Not sure what to set the default as though.